Ginger/GingerPluginCore/RecordingLib/PageChangedEventArgs.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Amdocs.Ginger.Plugin.Core
+{
+    public class PageChangedEventArgs
+    {
+        public string PageURL { get; set; }
```

Comment:
need to change the class and file name to "RecordedPageChangedEventArgs"

Reply:
renamed